tmva/sofie/README.md

```diff
@@ -18,57 +18,58 @@ Build ROOT with the cmake option tmva-sofie enabled.
 cmake ../root -Dtmva-sofie=ON
 make -j8
 ```
-    
+
 ## Usage
 SOFIE works in a parser-generator working architecture. With SOFIE, the user gets an [ONNX](https://github.com/root-project/root/tree/master/tmva/sofie_parsers), [Keras](https://github.com/root-project/root/blob/master/tmva/pymva/src/RModelParser_Keras.cxx) and a [PyTorch](https://github.com/root-project/root/blob/master/tmva/pymva/src/RModelParser_PyTorch.cxx) parser for translating models in respective formats into SOFIE's internal representation.
 
 From ROOT command line, or in a ROOT macro, we can proceed with an ONNX model:
 
-	using namespace TMVA::Experimental;
-	SOFIE::RModelParser_ONNX parser;
-	SOFIE::RModel model = parser.Parse(“./example_model.onnx”);
-	model.Generate();
-	model.OutputGenerated(“./example_output.hxx”);
+    using namespace TMVA::Experimental;
+    SOFIE::RModelParser_ONNX parser;
+    SOFIE::RModel model = parser.Parse(“./example_model.onnx”);
+    model.Generate();
+    model.OutputGenerated(“./example_output.hxx”);
```

Comment:
```suggestion
```c++
using namespace TMVA::Experimental;
SOFIE::RModelParser_ONNX parser;
SOFIE::RModel model = parser.Parse(“./example_model.onnx”);
model.Generate();
model.OutputGenerated(“./example_output.hxx”);
```
```

Why not use these three apostrophes? Then you don't have to worry about indentation, and can even use syntax highlighting (also in the other places in this file)